src/components/node/Node.jsx

```diff
@@ -41,83 +41,88 @@ import nodeHelper from './node.helper';
  *     onMouseOutNode={onMouseOutNode} />
  */
 export default class Node extends React.Component {
-    /**
-     * Handle click on the node.
-     * @returns {undefined}
-     */
-    handleOnClickNode = () => this.props.onClickNode && this.props.onClickNode(this.props.id);
+  /**
+   * Handle click on the node.
+   * @returns {undefined}
+   */
+  handleOnClickNode = () => this.props.onClickNode && this.props.onClickNode(this.props.id);
 
-    /**
-     * Handle mouse over node event.
-     * @returns {undefined}
-     */
-    handleOnMouseOverNode = () => this.props.onMouseOverNode && this.props.onMouseOverNode(this.props.id);
+  /**
+   * Handle mouse over node event.
+   * @returns {undefined}
+   */
+  handleOnMouseOverNode = () => this.props.onMouseOverNode && this.props.onMouseOverNode(this.props.id);
 
-    /**
-     * Handle mouse out node event.
-     * @returns {undefined}
-     */
-    handleOnMouseOutNode = () => this.props.onMouseOut && this.props.onMouseOut(this.props.id);
+  /**
+   * Handle mouse out node event.
+   * @returns {undefined}
+   */
+  handleOnMouseOutNode = () => this.props.onMouseOut && this.props.onMouseOut(this.props.id);
 
-    render() {
-        const nodeProps = {
-            cursor: this.props.cursor,
-            onClick: this.handleOnClickNode,
-            onMouseOut: this.handleOnMouseOutNode,
-            onMouseOver: this.handleOnMouseOverNode,
-            opacity: this.props.opacity
-        };
+  render() {
+    const nodeProps = {
+      cursor: this.props.cursor,
+      onClick: this.handleOnClickNode,
+      onMouseOut: this.handleOnMouseOutNode,
+      onMouseOver: this.handleOnMouseOverNode,
+      opacity: this.props.opacity
+    };
 
-        const textProps = {
-            dx: this.props.dx || CONST.NODE_LABEL_DX,
-            dy: CONST.NODE_LABEL_DY,
-            fontSize: this.props.fontSize,
-            fontWeight: this.props.fontWeight,
-            opacity: this.props.opacity
-        };
+    const textProps = {
+      dx: this.props.dx || CONST.NODE_LABEL_DX,
+      dy: CONST.NODE_LABEL_DY,
+      fontSize: this.props.fontSize,
+      fontWeight: this.props.fontWeight,
+      fill: this.props.fontColor,
```

Comment:
Prettier ran and formatted this file so it's hard to tell, bu this is the main change in this file (the addition of fill on textProps)

Reply:
Regarding the indentation can you please change the file `react-d3-graph/.prettierrc.js`? It has a typo 😄 , `tabWidt` should be `tabWidth: 4`, this should make prettier reindent the file correctly. Thanks!